src/main/java/bio/terra/service/dataset/DatasetRequestValidator.java

```diff
@@ -11,17 +11,22 @@
 import bio.terra.model.DatasetRequestModel;
 import bio.terra.model.DatasetSpecificationModel;
 import bio.terra.model.TableModel;
+import bio.terra.model.TableDataType;
+
 import bio.terra.common.ValidationUtils;
 import org.springframework.stereotype.Component;
 import org.springframework.validation.Errors;
 import org.springframework.validation.Validator;
 
 import javax.validation.constraints.NotNull;
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.HashMap;
 import java.util.HashSet;
 import java.util.List;
 import java.util.Optional;
 
+import static java.util.stream.Collectors.joining;
```

Comment:
I think in general we want to avoid `static` imports in our code since it makes it harder to tell where a name is coming from when reading the code. The exceptions this rule are the use of hamcrest matchers and related assert methods in test code. So I think the original static import of `toList()` was a mistake.